lifecycle/README.rst

```diff
@@ -256,7 +248,7 @@ It returns a list of all possible transitions this node can execute.
 ros2 lifecycle command line interface
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ``lifecycle_service_client`` application is a fixed order script for this demo purpose only.
+The ``lifecycle_service_client`` application is a fixed order script for demo purposes only.
```

Comment:
@clalancette I understand what it means by _fixed order_, but perhaps it can be phrased better?